lightning/src/blinded_path/utils.rs

```diff
@@ -51,10 +51,8 @@ macro_rules! build_keys_helper {
 						hmac.input(encrypted_data_ss.as_ref());
 						Hmac::from_engine(hmac).to_byte_array()
 					};
-					pk.mul_tweak(
-						$secp_ctx,
-						&Scalar::from_be_bytes(hop_pk_blinding_factor).unwrap(),
-					)?
+					pk.mul_tweak($secp_ctx, &Scalar::from_be_bytes(hop_pk_blinding_factor).unwrap())
```

Comment:
Do you know what the original reason was for making an exception here and not panicking?